boards/dactyl_manuform/readme.md

```diff
@@ -0,0 +1,49 @@
+# Dactyl Manuform
+
+The [Dactyl Manuform](https://github.com/tshort/dactyl-keyboard) is a handwired, split bodied, tented, curved keywell keyboard - adapted from the [Dactyl](https://github.com/adereth/dactyl-keyboard) keyboard with the thumb cluster design of the [ManuForm](https://geekhack.org/index.php?topic=46015.0) keyboard. Information for building a Dactyl Manuform found in first link.
+
+![Imgur](https://i.imgur.com/7y0Vbyd.jpg)
+*Two wired Dactyl Manuform 4x6 variants*
+
+## Variants
+
+Dactyl Manuform's are built in variations that cater for different row and column counts, and thumb clusters.  
+
+As standard: 
+- The finger keywell bottom row has 2 keys, 1 each in ring and middle columns
+    - Exception to this rule is the 7 column variants, that have two additional keys in this row
+- The thumb cluster has 6 keys, arranged in 2 columns by 3 rows 
```

Comment:
```suggestion
V ariants are denoted as `RowCount`*x*`ColumnCount` and share the common configuration of:
- The finger key-well bottom row has 2 keys, 1 each in ring and middle columns. Exception to this rule is the 7 column variants that have two additional keys in this row.
- The thumb cluster has 6 keys, arranged in 2 columns by 3 rows .
```

Is "keywell" the proper spelling? It looks like a compositum that should have a hyphen, but I'm not a linguist.